MAINTAINERS.MD

```diff
@@ -1,3 +1,6 @@
 | Org                    | Name                                                |
 | -----------------------| ----------------------------------------------------|
```

Comment:
Please add the GitHub user names according to the template (I need them to invite to @camaraproject/device-swap_maintainers).

```suggestion
| Org                    | Name                                      | GitHub Username        |
| -----------------------| ------------------------------------------| -----------------------|
```